Dockerfile.dev

```diff
@@ -27,6 +27,16 @@ RUN useradd -ms /bin/bash -l -u $USER_ID $USER_NAME && \
     echo "%sudo ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers && \
     echo "source /usr/local/lib/bazel/bin/bazel-complete.bash" >> /home/$USER_NAME/.bashrc
 
+RUN apt-get update && apt-get install -y python3 python3-pip \
+    gnupg wget software-properties-common
+
+RUN pip3 install prompt_toolkit jinja2 clang
+
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+RUN apt-add-repository "deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch-6.0 main"
+RUN apt-get update && apt-get install -y libclang-6.0-dev
```

Comment:
We already installed clang in the based build image (Dockerfile.build). You can specify the clang library when you invoke the main.py script

Reply:
Yes, I saw that you have already installed clang in(Dockerfile.build) but module (https://pypi.org/project/clang/) that is required by the "main.py" script expects clang with concrete version(up to 6.0.0, https://github.com/ethanhs/clang/issues/4)